src/main/java/org/unice/model/Client.java

```diff
@@ -1,60 +1,24 @@
 package org.unice.model;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.*;
 import org.unice.model.submodel.Address;
 
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
 
 @Getter
 @Setter
 @Data
```

Comment:
Based on: https://projectlombok.org/features/Data
"@Data - A shortcut for @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor!"
I think only the @data decorator might be enough out of those 3 :)

Reply:
Ok, i'll correct it. Thanks